app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java

```diff
@@ -880,7 +935,7 @@ void shareToNextCloud(){
             phimpmeProgressBarHandler.show();
 
         } else {
-            Toast.makeText(this, "Please sign in to nextcloud from account manager"
+            Toast.makeText(this, "Please sign in to " + account + " from account manager"
```

Comment:
Using a snackbar here would be good I think as we are using it for other accounts.